src/System.Diagnostics.PerformanceCounter/src/System/Diagnostics/PerformanceCounterLib.cs

```diff
@@ -14,7 +14,7 @@
 
 using static Interop.Advapi32;
 
-#if !netcoreapp
+#if !netcoreapp || netcoreapp20
```

Comment:
Are these ORs needed?  `netcoreapp` says it's set if the target group starts with "netcoreapp", and `netcoreapp20` is set if the target group is "netcoreapp2.0".  Since "netcoreapp2.0".StartsWith("netcoreapp") => true, I don't think `netcoreapp20` can be set without `netcoreapp` also being set.  Right?

Reply:
It is necessary because it excludes `netcoreapp20` from `netcoreapp`  since apparently `MemoryMarshal` is new from 2.1.

Reply:
Right.  Because there's a `!` there.  Oops.